osf/models/analytics.py

```diff
@@ -126,7 +126,12 @@ def update_counter(cls, page, node_info):
             # After we're sure this is stable, we can stop writing to the _id field, and query on
             # resource/file/action/version
             resource, file, action, version = cls.deconstruct_id(cleaned_page)
-            model_instance, created = cls.objects.select_for_update().get_or_create(_id=cleaned_page, resource=resource, file=file, action=action, version=version)
+            model_instance, created = PageCounter.objects.select_for_update().get_or_create(_id=cleaned_page)
+
+            model_instance.resource = resource
+            model_instance.file = file
+            model_instance.action = action
+            model_instance.version = version
```

Comment:
Debated whether to only populate these if `created` is True, and let the management command catch the old records over time, or populate them regardless.

Reply:
@brianjgeiger - should I only set these if `created` is True?

Reply:
discussed in person, setting them either way, leaving as-is.